### PR TITLE
remove enums

### PR DESCRIPTION
**Pull request by shintasmith**
Remove enums implementation and support

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi @ChandraAddala @izrik @drenalin23 pls review

**Comment by shintasmith** [comment]
Oh, btw, I need someone who knows about how Enums are implemented in blueflood-elasticsearch to scrutinize the code changes there. I have to admit, I'm shooting in the dark! 😬

**Comment by ChandraAddala** [comment]
The only comment I have is, we havent removed column family definitions of column families corresponding to enums.

File: blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java [hunk 1]
```diff
@@ -142,83 +133,16 @@
         //token paths matching query, which also have a next level.
         tokenInfoBuilder.addTokenPathWithNextLevel(metricIndexData.getTokenPathsWithNextLevel());
 
-        Set<String> completeMetricsMatchingQueryPrevLevel = metricIndexData.getCompleteMetricNamesAtBaseLevel();
```
Comment (ChandraAddala) [inline blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/AbstractElasticIO.java]: We need to know if it is a complete metric name at base level to look for enums. If there are no enums we dont need it.

But I think we can adjust the regex expression a little bit, now that we dont need that information. We can create another story to look into this API after we cleanup enums.

File: blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/MetadataIOIntegrationTest.java [hunk 2]
```diff
@@ -69,7 +69,7 @@ public void writeAllAstyanaxReadAllDatastax() throws IOException {
 
         Table<Locator, String, String> meta = HashBasedTable.create();
         meta.put( l0, CACHE_KEY, RollupType.GAUGE.toString() );
-        meta.put( l1, CACHE_KEY, RollupType.ENUM.toString() );
+        meta.put( l1, CACHE_KEY, RollupType.SET.toString() );
```
Comment (shintasmith) [inline blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/MetadataIOIntegrationTest.java]: We are not adding it. I'm just replacing ENUMs with SETs.

**Pull request merged**